packages/components/src/components/button/button.jsx

```diff
@@ -61,14 +62,24 @@ const Button = ({
             {...props}
```

Comment:
We need `renderText` to follow our coding style regarding camelCasing props that are functions. But since it is being passed down in `{...props}`, react will throw a warning. Let's include this in prop de-structure section, so `{...props}` does not include this.